### PR TITLE
Ability to customize cache path and error message when not found added and readme updated - Fix #1

### LINKED ISSUE #1: Add ability to customize cache path and error message when not found
Mainly so it can be reused in [`alfy`](https://github.com/sindresorhus/alfy), but others might have a need for this too. I'm thinking environment variables, but happy to consider better ideas.
## 

// @mafredri

### PR DESCRIPTION
**Pull request by joaoppsantos**
Fix #1:
* Environment variables - `RUN_NODE_ERROR_MSG` and `RUN_NODE_CACHE_PATH` - added;
* Updated readme.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ This Bash script uses some tricks to find the Node.js binary on your system and
 
 Can be used from any environment that can spawn a process (Shell, Python, Ruby, Swift, Objective-C, etc).
 
+Error message for when node ain't found can be personalized by setting the environment variable "RUN_NODE_ERROR_MSG".
+Node cache path can be customized by adding the environment variable "RUN_NODE_CACHE_PATH".
```
Comment (sindresorhus) [inline readme.md:12]: This doesn't explain what the "Node cache path" is. It's unclear what's being cached.

File: readme.md [hunk 2]
```diff
@@ -8,6 +8,9 @@ This Bash script uses some tricks to find the Node.js binary on your system and
 
 Can be used from any environment that can spawn a process (Shell, Python, Ruby, Swift, Objective-C, etc).
 
+Error message for when node ain't found can be personalized by setting the environment variable "RUN_NODE_ERROR_MSG".
```
Comment (sindresorhus) [inline readme.md:11]: This needs to be written more formally. "Ain't" is not a good word to use in documentation. And it's `Node.js`, not node, or Node.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: readme.md (hunk 2 above)
Comment (joaoppsantos) [reply readme.md:11]: Is the following sentence more appropriate ?

Ìn case there is preference for a personalized error message (e.g. `Node.js` is not found in the system), you can have one by setting the environment variable "RUN_NODE_ERROR_MSG".

File: readme.md (hunk 1 above)
Comment (joaoppsantos) [reply readme.md:12]: This makes it more explicit ? 

In the other hand, if your desire is to customize the cache (temporary storage) path for the `Node.js` binary, you can do it by adding the environment variable "RUN_NODE_CACHE_PATH".

**Comment by joaoppsantos** [comment]
Sorry the time I took to answer. Had the comments in Pending status and only noticed it now, not very familiarized with Github yet.

**Comment by sindresorhus** [comment]
Closing in favor of #4

**Pull request closed without merging**